src/github.com/getlantern/flashlight/client/chained.go

```diff
@@ -96,6 +118,7 @@ func (s *ChainedServerInfo) Dialer(deviceID string) (*balancer.Dialer, error) {
 				return nil, err
 			}
 
+			s.addCheckTarget(addr)
```

Comment:
Just because we were able to dial the proxy doesn't mean that this is a good check target.  I don't think it matters that much, but wanted to make sure that's clear.